### PR TITLE
Updates entities.json5 re: RapidAPI

### PR DESCRIPTION
**Pull request by rickydazla**
See rapidapi.com — _"An API Marketplace"_

### CODE REVIEW

File: data/entities.json5 [hunk 1]
```diff
@@ -7523,6 +7523,11 @@
     "domains": ["*.movementventures.com", "*.qzzr.com"],
     "examples": ["www.qzzr.com"]
   },
+  {
+    "name": "RapidAPI",
+    "categories": ["utility"],
+    "domains": ["*.rapidapi.com"]
```
Comment (patrickhulce) [inline data/entities.json5:7529]: any chance you've got an example? like `cdn.rapidapi.com` or just generally how these urls look? :)

Comment (rickydazla) [reply data/entities.json5:7529]: @patrickhulce they advertise "a directory of some 8,000 APIs". I only use one, called Telize, for geolocation and the URL looks like this, but I couldn't speak to all the others: `telize-v1.p.rapidapi.com`

Comment (rickydazla) [reply data/entities.json5:7529]: I can add that to the PR if you think it's worthwhile?

Comment (patrickhulce) [reply data/entities.json5:7529]: yep that'd be great :)

Comment (rickydazla) [reply data/entities.json5]: done https://github.com/patrickhulce/third-party-web/pull/101/commits/794adbd0843c749393c1b461987f109c73f49f8b

Comment (patrickhulce) [reply data/entities.json5]: thanks!

**Pull request merged**